pkg/cmd/render/render.go

```diff
@@ -643,3 +670,24 @@ func getInfrastructure(file string) (*configv1.Infrastructure, error) {
 	}
 	return config, nil
 }
+
+func getBootstrapScalingStrategy(installConfig map[string]interface{}, delayedHAMarkerFile string) (ceohelpers.BootstrapScalingStrategy, error) {
+	// Delayed HA strategy is set if marker file exists on disk.
+	if _, err := os.Stat(delayedHAMarkerFile); err == nil {
+		return ceohelpers.DelayedHAScalingStrategy, nil
+	}
+	if _, found := installConfig["controlPlane"]; !found {
```

Comment:
Maybe unmarshall installConfig to its golang struct to ensure type safety?

Reply:
yeah, the entire render should get reworked.

Reply:
actually looking at this I will clean it up a bit but we would need to import the installer because it is not an actual openshift/api. This feels sort of dirty to me, we can talk about it as a followup.